test/py/test_gpu_async.py

```diff
@@ -23,10 +23,17 @@
 #####################################################################################
 import migraphx
 import ctypes
-
+import os
 
 def test_conv_relu():
-    hip = ctypes.cdll.LoadLibrary("libamdhip64.so")
+    library = "libamdhip64.so"
+
+    # Full path of the library is specified to fix
+    # an issue on sles where the library is not loaded otherwise.
+    if os.path.exists("/opt/rocm/lib/libamdhip64.so"):
+        library = "/opt/rocm/lib/libamdhip64.so"
```

Comment:
This could check first if `/opt/rocm` exists or not. 

If not then can check if `/opt/rocm-*` exists or not.

Reply:
Done.